libs/fonts/graphite/stone.yml

```diff
@@ -0,0 +1,35 @@
+#
+# SPDX-FileCopyrightText: © 2020-2022 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : graphite
+version     : '1.3.14'
+release     : 1
+homepage    : https://graphite.sil.org
+upstreams   :
+    - https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz : f99d1c13aa5fa296898a181dff9b82fb25f6cc0933dbaa7a475d8109bd54209d
+summary     : Allows the creation of fonts with complex behaviors
+description : |
+    Graphite is a system that can be used to create “smart fonts” capable of displaying writing systems with various complex behaviors. A smart font contains not only letter shapes but also additional instructions indicating how to combine and position the letters in complex ways.
+
+    Graphite was primarily developed to provide the flexibility needed for minority languages which often need to be written according to slightly different rules than well-known languages that use the same script.
+license     :
+    - LGPL-2.1-or-later
+builddeps   :
+    - gcc-devel
+checkdeps   :
+    - binary(readelf)
+setup       : |
+    %cmake
+build       : |
+    %cmake_build
+install     : |
+    %cmake_install
+check       : |
+    ninja -C %(builddir) test
+packages    :
+    - "%(name)-devel":
+        paths:
+            # Use a macro when https://github.com/serpent-os/moss-format/issues/2 is fixed.
```

Comment:
So I think the real issue is we need to have `*.cmake` files in `-devel` regardless. I don't think we're going to do path expansion in paths.